qa/integration/src/test/resources/features/job/JobExecutionServiceI9n.feature

```diff
@@ -9,9 +9,10 @@
 # Contributors:
 #     Eurotech - initial API and implementation
 ###############################################################################
-@integration
 @jobs
 @jobExecutionService
+@integration
+
```

Comment:
Same comment as above; if you wanted to put "integration" tags in the end it's ok, otherwise put it back.

Reply:
Hi @LeoNerdoG,

The position of integration tag is set to be last in every .feature file. If this is ok maybe I can keep this changes.